pimd/pimd.h

```diff
@@ -90,8 +90,8 @@
 #define PIM_MASK_PIM_PACKETDUMP_RECV (1 << 4)
 #define PIM_MASK_PIM_TRACE           (1 << 5)
 #define PIM_MASK_PIM_TRACE_DETAIL    (1 << 6)
-#define PIM_MASK_GM_EVENTS           (1 << 7)
-#define PIM_MASK_GM_PACKETS          (1 << 8)
+#define PIM_MASK_GM_EVENTS	     (1 << 7)
```

Comment:
is this patch even needed given the first patch has a bunch of white space changes